pkgs/data/themes/plata/default.nix

```diff
@@ -42,13 +40,12 @@ stdenv.mkDerivation rec {
   ++ lib.optionals mateSupport [ gtk3 marco ]
   ++ lib.optional telegramSupport zip;
 
-  buildInputs = [
-    gdk-pixbuf
-    librsvg
+  # GTK2 engines must be on the system path at runtime to be loaded.
+  propagatedUserEnvPkgs = [
+    gtk2
```

Comment:
afaik only gtk-engine-murrine needs to be in propagatedUserEnvPkgs, if so gtk2 should be in buildInputs

Reply:
And probably `gtk_engines` instead of `gtk2`.